index.html

```diff
@@ -4,9 +4,61 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-    <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" href="/css/index.css" />
+    <link href="https://unpkg.com/boxicons@2.0.9/css/boxicons.min.css" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@400;700&family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    
+    <section class="header">
+      <div class="nav">
+        <ul class="leftfloat">
+          <li>
+            <a class="button" href="/index.html"><img src="./img/mrtaaj.png" alt="mrtaaj"/></a>
+          </li>
+        </ul>
+        <ul class="rightfloat">
+          <li>
+            <a class="hamburger-button" href="">
+              <i class="bx bx-menu boxi-icon"></i>
+            </a>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section class="main">
+      <div class="main-flex">
+        <h1>Hey There. <br />I’m Tajudeen</h1>
+        <h4>I am a Software Developer</h4>
+        <p>
+          I can help you build a product , feature or website Look through some of my work and
+          experience! If you like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <div class="social">
+          <a class="button" href="">
+            <i class="bx bxl-github soc-icon"></i>
+          </a>
+          <a class="button" href="">
+            <i class="bx bxl-linkedin soc-icon"></i>
+          </a>
+          <a class="button" href="">
+            <i class="bx bxl-twitter soc-icon"></i>
+          </a>
+          <a class="button" href="">
+            <i class="bx bxl-medium soc-icon"></i>
+          </a>
+        </div>
+      </div>
+      <div class="abs-back">
+        <img src="./img/main-background.png" alt="" />
```

Comment:
We should use a background here as this represents a background instead of a normal image `<img>` tag

Reply:
Thank you for the corrections; I will take note of everything. In this case, I think it is best if it is an image because it has to be placed at a specific place in this section. Perhaps my naming for the image is making it confusing.

Reply:
Cool. Well It's not actually about the placement but rather the concept itself. When we need to have other things that a user will focus on above an image it's the best practice to use a background but if the image is the main focus the best practice would be to use an `<img>` tag i.e: e-commerce products image or teams image.

And you are welcome

Reply:
Okay. Noted and well understood.